Fix handling of session files with `Cookie:` followed by other headers (#1127)
httpie/httpie: CHANGELOG.rst

  an alternative to ``stdin``. (`#534`_)
* Fixed ``--continue --download`` with a single byte to be downloaded left. (`#1032`_)
* Fixed ``--verbose`` HTTP 307 redirects with streamed request body. (`#1088`_)

> * Fixed session files handling when there are cookies and other headers following. (`#1126`_)

Reviewer: ```suggestion
* Fixed handling of session files with `Cookie:` followed by other headers. (`#1126`_)
```
